code/API_definitions/connected-network-type.yaml

```diff
@@ -0,0 +1,405 @@
+openapi: 3.0.3
+info:
+  title: Connected Network Type
+  description: |
+    This API provides the customer with the ability to query to which Mobile Communication Technology it is connected to.
+
+    # Introduction
+
+    ## Connected Network Type
+
+    API consumer is able to inquire device's connected network type.
+
+    # Relevant terms and definitions
+
+    * **Device**: A device refers to any physical entity that can connect to a network and participate in network communication.
+      At least one identifier for the device (user equipment) out of four options: IPv4 address, IPv6 address, Phone number, or Network Access Identifier assigned by the mobile network operator for the device.
+
+    * **Network Type** : Network Type is intended to provide insight to connected network's capabilities from standards perspective.  Actual network capabilities may differ based on implementation and MUST be checked with the connected network provider.
+      - `2G`, if device is connected to the 2G network technology
+      - `3G`, if device is connected to the 3G network technology
+      - `4G`, if device is connected to the 4G network technology
+      - `5G`, if device is connected to the 5G network technology
+      - `UNKNOWN` if connection [technology] can not be determined
+
+
+    * **LastStatusTime** : This property specifies the time when the status was last updated. Its presence in the response indicates the freshness of the information, while its absence implies the information may be outdated or its freshness is uncertain.
+
+    # API Functionality
+
+    The API exposes following capabilities:
+
+    ## Connected Network Type
+
+    The endpoint `POST /retrieve-network-type` allows to get connected Network Type.
+
+
```

Comment:
If we allow device identifier in the access token, we need to add the template to the description: https://github.com/camaraproject/Commonalities/blob/main/documentation/API-design-guidelines.md#appendix-a-normative-infodescription-template-for-when-user-identification-can-be-from-either-an-access-token-or-explicit-identifier